features/support/env.rb

```diff
@@ -41,7 +41,7 @@ def run_xcpretty(flags)
   File.open(input_file.path, 'w') do |file|
     file.print run_input
   end
-  @run_output = %x(cat '#{input_file.path}' | bin/xcpretty #{flags})
+  @run_output = %x(cat '#{input_file.path}' | bin/xcpretty #{flags} 2>&1)
```

Comment:
Why do you redirect `stderr` to `stdout`?

Reply:
Not needed as the test is not removed, so this change is reverted.

Reply:
But you didn't revert it...

Reply:
done